components/aws/sagemaker/codebuild/integration-test.buildspec.yml

```diff
@@ -7,18 +7,17 @@ env:
 phases:
   build:
     commands:
-      - cd components/aws
-      - docker build . -f ./sagemaker/tests/integration_tests/Dockerfile -t amazon/integration-test-image --quiet
+      - docker build . -f ./components/aws/sagemaker/tests/integration_tests/Dockerfile -t amazon/integration-test-image --quiet
```

Comment:
what has changed here ?

Reply:
context of docker build changed to repo's root directory

Reply:
build context of docker is different 

it used to be "components/aws"
now it is the root of the git repo